### DISCUSSION TITLE
Error running OpenFAST from MATLAB

### DISCUSSION THREAD

**Discussion by andreowhite** [OpenFAST/openfast#1092] [General]
Hi everyone:
I am trying to run OpenFAST from MATLAB and get the following error:
**FAST_InitializeAll:FAST_Init:FAST_ReadPrimaryFile:Invalid numerical input for file** 
**"5MW_OC4Semi_WSt_WavesWN.fst" occurred while trying to read MHK.** 
 
** OpenFAST encountered an error during module initialization. 
 Simulation error level: FATAL ERROR** 
 
 **Aborting OpenFAST.** 

Does anyone know how to solve this problem?
Am I missing a file from the controller why it is unable to read MHK?

The command that I use to run simulation was:
**dos(['openfast ', fname]) fname is the respective .fst file .**

Thank you.

Regards,
AOAW

**Comment by jjonkman** [comment]
Hi @andreowhite,

It looks like you asked the same question on our forum: https://forums.nrel.gov/t/how-do-i-run-openfast/2425/32. I've now responded there.

Best regards,

**Reply by andreowhite** [reply]
Yes @jjonkman that was the case. Many thanks.

Regards,
AOAW
